include/psen_scan/sync_udp_writer.h

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2020 Pilz GmbH & Co. KG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#ifndef PSEN_SCAN_SYNC_UDP_WRITER_H
+#define PSEN_SCAN_SYNC_UDP_WRITER_H
+
+#include <stdexcept>
+#include <array>
+#include <string>
+
+#include <boost/asio.hpp>
+
+#include "psen_scan/open_connection_failure.h"
+#include "psen_scan/close_connection_failure.h"
+
+namespace psen_scan
+{
+class WriteFaliure : public std::runtime_error
```

Comment:
Because `OpenConnectionFailure` is used by two classed (the async UDP reader and the sync writer) whereas `WriteFaliure` is only used by the sync writer.